_posts/0000-01-04-bobbiemckeeTN.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Go go Roboto Cat
```

Comment:
Dunder Mifflin Paper Co.

Reply:
People say I'm the best boss.

![1500-0-](https://user-images.githubusercontent.com/38323656/80406572-e3aec400-8889-11ea-811c-97b473cb5ec5.png)